hapic/context.py

```diff
@@ -206,8 +206,9 @@ def decorator(*args, **kwargs):
                             exc,
                             include_traceback=self.is_debug(),
                         )
+                        dumped = error_builder.dump(error_body).data
```

Comment:
This error should be checked if it exist in another context.

Reply:
ok